core/domain/exp_domain.py

```diff
@@ -621,6 +622,25 @@ def __init__(
         self.auto_tts_enabled = auto_tts_enabled
         self.correctness_feedback_enabled = correctness_feedback_enabled
 
+    def get_translatable_contents_collection(
+        self
+    ) -> translation_domain.TranslatableContentsCollection:
+        """Get all translatable fields/objects in the exploration.
```

Comment:
```suggestion
        """Get all translatable fields in the exploration.
```
Ditto elsewhere (now we only return translatable fields and not objects)